### PR TITLE
feat: show 'project group' in cost analysis, dashboard

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [x] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA

### CODE REVIEW

File: apps/web/src/services/cost-explorer/constants/cost-explorer-constant.ts [hunk 1]
```diff
@@ -26,6 +26,7 @@ export const ADDITIONAL_GROUP_BY_ITEM_MAP = {
 } as const;
 
 export const GROUP_BY_ITEM_MAP = {
+    [GROUP_BY.PROJECT_GROUP]: { name: GROUP_BY.PROJECT_GROUP, label: 'Project Group' },
```
Comment (yuda110) [inline apps/web/src/services/cost-explorer/constants/cost-explorer-constant.ts:29]: 스크린샷 2024-03-12 오후 1 48 55

File: apps/web/src/services/dashboards/widgets/_constants/widget-options-schema-constant.ts [hunk 2]
```diff
@@ -38,6 +38,15 @@ export const WIDGET_FILTERS_SCHEMA_PROPERTIES: Record<WidgetFilterOptionKey, Wid
...
     },
+    'filters.project_group': {
+        key: MANAGED_VARIABLE_MODEL_CONFIGS.project_group.key,
+        name: MANAGED_VARIABLE_MODEL_CONFIGS.project_group.name,
+        selection_type: 'MULTI',
+        inheritance_mode: 'KEY_MATCHING',
+        item_options: [
+            { type: 'MANAGED', key: MANAGED_VARIABLE_MODEL_CONFIGS.project_group.key },
+        ],
+    },
```
Comment (yuda110) [inline apps/web/src/services/dashboards/widgets/_constants/widget-options-schema-constant.ts:49]: 스크린샷 2024-03-12 오후 1 50 00

File: apps/web/src/services/dashboards/constants/managed-variables-schema.ts [hunk 3]
```diff
@@ -16,6 +16,17 @@ export const MANAGED_DASH_VAR_SCHEMA: DashboardVariablesSchema = {
...
+            name: MANAGED_VARIABLE_MODEL_CONFIGS.project_group.name,
+            variable_type: 'MANAGED',
+            use: true,
+            selection_type: 'MULTI',
+            description: 'DASHBOARDS.CUSTOMIZE.VARIABLES.DESCRIPTION_PROJECT_GROUP', // TODO: Add description
+            options: [{
+                type: 'MANAGED',
+                key: MANAGED_VARIABLE_MODEL_CONFIGS.project_group.key,
+            }],
+        },
```
Comment (yuda110) [inline apps/web/src/services/dashboards/constants/managed-variables-schema.ts:29]: 스크린샷 2024-03-12 오후 1 49 39

File: apps/web/src/services/dashboards/dashboard-template/templates/cdn-and-traffic-cost.ts [hunk 4]
```diff
@@ -24,6 +24,7 @@ const widgetList: Parameters<typeof getDashboardLayoutWidgetInfoList>[0] = [
             'cost_data_field',
             'cost_data_type',
             'filters.cost_product',
+            'filters.project_group',
             'filters.project',
```
Comment (piggggggggy) [inline apps/web/src/services/dashboards/dashboard-template/templates/cdn-and-traffic-cost.ts:28]: There might be some conflict...

**Review by piggggggggy — APPROVED** [review]
LGTM~!

**Pull request merged**